index.js

```diff
@@ -65,8 +64,43 @@ function MLLPServer(host, port) {
 
     });
 
-    Server.listen(PORT, HOST);
+    self.send = function (receivingHost, receivingPort, hl7Data, callback) {
+        var sendingClient = new net.connect({
+            host: receivingHost,
+            port: receivingPort
+        }, function () {
+            console.log('Sending data to ' + receivingHost + ':' + receivingPort);
+            sendingClient.write(VT + hl7Data + FS + CR);
+        });
+
+        sendingClient.on('data', function (rawAckData) {
+            console.log(receivingHost + ':' + receivingPort + ' ACKED data');
+
+            var ackData = rawAckData
+                .toString() // Buffer -> String
+                .replace(VT, '')
+                .split('\r')[1] // Ack data
+                .replace(FS, '')
+                .replace(CR, '');
+
+            callback(null, ackData);
```

Comment:
If we have an AE message, is it not preferable to consider it as an error? True question here, I really don't know what's the best...